### DISCUSSION TITLE
Connection to Gemini

### DISCUSSION THREAD

**Discussion by analyticsrepo01** [vanna-ai/vanna#288] [General]
Hi Team
Are you open to connecting Gemini Pro model 
is there anything needed to support that?

**Comment by zainhoda** [comment]
@analyticsrepo01 We'd love Gemini support. When we started looking into it, we couldn't find how to set the Gemini system prompts.

There are some suggestions here to potentially try to shoehorn our prompt construction into a format that Gemini will accept:
https://www.googlecloudcommunity.com/gc/AI-ML/Implementing-System-Prompts-in-Gemini-Pro-for-Chatbot-Creation/m-p/712132
